src/auro-flight-main.js

```diff
@@ -6,20 +6,22 @@
 // If use litElement base class
 import { LitElement, html, css } from "lit-element";
 import styleCss from "./style-flight-main-css.js";
+import { getDateDifference } from './../util/util';
 
 // See https://git.io/JJ6SJ for "How to document your components using JSDoc"
 /**
  * The auro-flight-main element renders the middle 'frame' of the auro-flight component with the auro-flightline.
  * DoT: STATION SIZE AND COLOR MUST BE IDENTICAL TO DISCLOSURE SIZE AND COLOR!
  *
+ * @attr {Array} stops - Array of objects representing stopovers or layovers: "isStopover": bool, "arrivalStation": string, "duration": string ["123hr 123m"] (layover only)
```

Comment:
This new attribute is only for a11y, right? This should be noted.

Reply:
Please update this description to state that this content will NOT be used in the UI, but only constructs the a11y conversational phrase.

Reply:
If we can get this resolved, we can ship this update.

Reply:
@blackfalcon updated